### PR TITLE
Create li2023Rethinking.markdown

### PR DESCRIPTION
**Pull request by Alex-HaochenLi**
Thank you for submitting a PR! Please replace this text with a high-level description of the PR. Also ensure the following for new publications:

- [x] Files for new publications are in the `_publications` folder.
- [x] The name of each file is `lastnameYEARfirstword.markdown`, _e.g._ `smith2019neural` for a Smith _et al._ paper title "A neural approach to the Universe".
- [x] Consider using tags that already exist. We aim to avoid variations or introducing new ones when possible. This is to help searching across this literature review.

### CODE REVIEW

File: _publications/li2023rethinking.markdown [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: publication
+title: Rethinking Negative Pairs in Code Search
+authors: Haochen Li, Xin Zhou, Luu Anh Tuan, Chunyan Miao
+conference: EMNLP
+year: 2023
+additional_links:
+   - {name: "ArXiV", url: "https://arxiv.org/abs/2310.08069"}
+   - {name: "code", url: "https://github.com/Alex-HaochenLi/Soft-InfoNCE"}
+tags: ["search", "Transformer", "retrieval", "optimization"]
```
Comment (bzz) [inline _publications/li2023rethinking.markdown:10]: How about adding a "representation" here as well?

Comment (bzz) [inline _publications/li2023rethinking.markdown:10]:
```suggestion
tags: ["search", "Transformer", "retrieval", "optimization", "representation"]
```

Comment (bzz) [reply _publications/li2023rethinking.markdown:10]: How about adding a "representation" here as well?

Comment (Alex-HaochenLi) [reply _publications/li2023rethinking.markdown]: Thank you for your suggestion. I click 'Commit suggestion' and I think this operation directly adds your modification to my pull request?

**Comment by Alex-HaochenLi** [comment]
@mallamanis Hello, are there any further issues with this PR?

**Comment by mallamanis** [comment]
Thanks for the contibution and your patience. This PR must have slipped through before.

**Pull request merged**